builder/docker.go

```diff
@@ -47,6 +47,7 @@ func (b *Builder) GetHostAndUpstreamFormats() (string, string, error) {
 	return string(hostFormat), upstreamFormat, nil
 }
 
+// GetDockerImageName gets the existing image name
```

Comment:
Did you plan on exporting this function and gave up?

Reply:
It was exported in a V2 :P